CMakeLists.txt

```diff
@@ -365,24 +372,52 @@ if(UMF_FORMAT_CODE_STYLE)
             COMMENT "Format Cmake files using cmake-format")
     endif()
 
-    # Add a convenience target for running both tools at once - available only
-    # if both are found.
-    if(CLANG_FORMAT AND CMAKE_FORMAT)
+    if(BLACK)
+        # black should maintain backward compatibility, we don't have to require
+        # a specific version
+        get_program_version_major_minor(${BLACK} BLACK_VERSION)
+        message(STATUS "Found black: ${BLACK} (version: ${BLACK_VERSION})")
+
+        message(
+            STATUS
+                "Adding 'black-format-check' and 'black-format-apply' make targets"
```

Comment:
`make targets` - does it mean that MSVC targets won't be created?

Reply:
Removed all `make` refs in these messages